### PR TITLE
Cloud 1908 applications

### PR DESCRIPTION
**Pull request by pedanticdev**
Documented large parts of application management. Please review, if good, merge and then I can pull and continue with the rest. I think we should merge so at least we have something on the main branch.

### CODE REVIEW

File: docs/modules/reference/pages/manage/app/upload.adoc [hunk 1]
```diff
@@ -1,5 +1,49 @@
...
+
+[checklist]
+* ❶ Binary Upload Button: Click to bring up the file upload dialog to pick an application binary (.WAR file).
+* ❷ Name Of Artefact: Shows the name of the selected application binary
+* ❸ Deploy Option: Option to directly deploy the uploaded binary
+* ❹ Application Name: Option to give the application a name distinct from the uploaded binary file
+
+An uploaded application as shown in the above dialog will transition to the xref:manage/app/detail.adoc#_pending_state[Pending State].
+Custom xref:manage/app/configuration/overview.adoc[configurations] can be carried out on the uploaded application to ready it for deployment.
+You can directly deploy the application right after uploading by clicking the Deploy Immediately option in ❸.
```
Comment (rohan267) [inline docs/modules/reference/pages/manage/app/upload.adoc:27]: `Application can be immediately deployed by selecting 'Deploy immediately' option and click Upload`

File: docs/modules/reference/pages/manage/app/upload.adoc [hunk 2]
```diff
@@ -1,5 +1,49 @@
...
+A directly deployed application transitions to the xref:manage/app/detail.adoc#_running_state[Running State]
```
Comment (rohan267) [inline docs/modules/reference/pages/manage/app/upload.adoc:28]: `will transit into`

File: docs/modules/reference/pages/manage/app/upload.adoc [hunk 3]
```diff
@@ -1,5 +1,49 @@
...
+
+== Deploying Pending Applications
+
+A pending application can be deployed by clicking the Deploy Changes menu option from the Application Actions drop-down menu
```
Comment (rohan267) [inline docs/modules/reference/pages/manage/app/upload.adoc:32]: A pending application can be deployed by clicking the 'Deploy Changes' menu option from the Application Actions drop-down menu

File: docs/modules/reference/pages/manage/app/upload.adoc [hunk 4]
```diff
@@ -1,5 +1,49 @@
...
+
+.Deploying an uploaded application
+image::manage/application/application-11-deploy.png[]
+
+[checklist]
+* ❶ Deploy Changes: Deploy the uploaded application, transitioning it to the  xref:manage/app/detail.adoc#_running_state[Running State]
+* ❷ and ❸ Both menus having the Deploy Changes option as well
```
Comment (rohan267) [inline docs/modules/reference/pages/manage/app/upload.adoc:39]: `having` can be replaced with `have`

**Review by rohan267 — CHANGES_REQUESTED** [review]

File: docs/modules/reference/pages/manage/app/upload.adoc (hunk 2 above)
Comment (rohan267) [reply docs/modules/reference/pages/manage/app/upload.adoc:28]: is this change remainin?

Comment (pedanticdev) [reply docs/modules/reference/pages/manage/app/upload.adoc:28]: No. Transition is the right work in this context --> https://wikidiff.com/transit/transition

**Review by rohan267 — APPROVED** [review]

**Pull request merged**